dash_cytoscape/package.json

```
@@ -48,34 +48,32 @@
     "cytoscape-spread": "^3.0.0",
     "cytoscape-svg": "0.4.0",
     "lodash": "^4.17.21",
-    "ramda": "^0.29.0",
-    "react": ">=16",
+    "ramda": "^0.29.1",
     "react-cytoscapejs": "2.0.0",
-    "react-dom": ">=16",
-    "uuid": "^9.0.0"
+    "uuid": "^9.0.1"
   },
   "devDependencies": {
-    "@babel/core": "^7.23.0",
-    "@babel/eslint-parser": "^7.22.15",
-    "@babel/preset-env": "^7.22.20",
-    "@babel/preset-react": "^7.22.15",
+    "@babel/core": "^7.23.7",
+    "@babel/eslint-parser": "^7.23.3",
+    "@babel/preset-env": "^7.23.8",
+    "@babel/preset-react": "^7.23.3",
     "babel-loader": "^9.1.3",
     "copyfiles": "^2.4.1",
-    "css-loader": "^6.8.1",
-    "eslint": "^8.50.0",
-    "eslint-config-prettier": "^9.0.0",
-    "eslint-plugin-import": "^2.28.1",
+    "css-loader": "^6.9.1",
+    "eslint": "^8.56.0",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-plugin-import": "^2.29.1",
     "eslint-plugin-react": "^7.33.2",
-    "npm": "^10.1.0",
-    "prettier": "^3.0.3",
+    "npm": "^10.3.0",
+    "prettier": "^3.2.4",
     "react-docgen": "^5.3.0",
-    "style-loader": "^3.3.3",
-    "webpack": "^5.88.2",
-    "webpack-cli": "^5.1.4"
+    "style-loader": "^3.3.4",
+    "webpack": "^5.89.0",
```

```suggestion
    "webpack": "^5.90.0",
```
hot off the presses 😉